### DISCUSSION TITLE
Air Temps and Water Temps in a fight for dominance on dashPanel

### DISCUSSION THREAD

**Discussion by gw8674** [tagyoureit/nodejs-poolController#622] [REM] [answered]
I've been fighting with inaccurate temps from the IntelliCenter thermistors for a while now.  Once calibrated, they get off pretty quickly.
Posted a new wish list item (deleted now) to be able to feed air temp and water temp from REM to dashPanel.

Then it came to me (light bulb moment), oh yea, I can do that through the "Device Value Feeds" from REM to njspC-Automatic feature. But then remembered that I had cancelled that ability because the two water temp values and air temp values were battling it out again.

Long ago it had done that same thing but @rstrouse came up with a PFM solution which fixed it. But I can't remember what we did. However about 6 months ago, the battles between the two air temp values and the two pool temp values came back to haunt me once again. Possibly after one of the pulls but I can't say for sure.

Anyway, when I saw the constant battles between the Atlas Air Temp feed and IntelliCenter Air Temp feed as well as the Atlas Water Temp Feed and Intellicenter Water Temp Feed, I just deleted those two Device Value Feeds from REM vowing to come back to this problem later.

Well, I guess later has arrived.

Any chance that I could get assistance with this issue?  It would be great to just somehow have the ability to "turn off" the IntelliCenter thermistor value feeds to the dashPanel and just have the two temp values from REM appear on the dashPanel without watching a boxing match between the two temp values. 🤣

Thanks very much guys!  Really appreciate all you do!!!
(I'll see if I can post a video as an example of what's happening but not sure github will allow it. Thanks...

https://user-images.githubusercontent.com/52094163/192400496-18f2c8fc-cdba-4d4d-9f96-ebfd05cdab74.mp4

**Comment by rstrouse** [comment, marked as answer]
We never would have attempted to override the OCP temps.  The problem is that the OCP uses those as gospel for Air, Solar, and Water.   You really can't for the OCP to ingest an external value.  Now if the wiring is good and the probe ohms out properly then I would go into the calibration and see if you can get it to clear its adjusted value.  Do this on the OCP screen itself and set the Adjusted value to the True Readout.  This should clear the +- degree settings.  Then set your calibration values there.

Initially I did this on mine from the Web App and fought with it a bit until I finally went to the OCP to set it.  You will find it no more or less accurate than the Atlas probe.  Voltage and resistance are voltage and resistance no matter how you measure it.  You are showing a difference of over 7 degrees.

**Comment by gw8674** [comment]
@rstrouse 
Good morning.
Understand about not being able to override the Pentair thermistor values in dashPanel.
Followed your instructions regarding clearing the adjusted values and setting the calibration values at the OCP instead of doing it from the web app.

Time will tell if the IntelliCenter calibrated values hold and that does the trick. 🤞 

I removed the two temp feeds from REM to njspC to eliminate any possible confliction in the future and will just rely on the two temp values provided by the IntelliCenter thermistors.

Thanks very much as always for the timely advice!
